src/middleware/builtin.ts

```diff
@@ -288,6 +288,11 @@ export function matchEventType(pattern: EventTypePattern): Middleware<SlackEvent
   };
 }
 
+// TODO: breaking change: why does this method have to be invoked as a function with no args, while other similar
+// method like the `only*` ones do not require that? should make this consistent.
```

Comment:
If we rename this to onlyNonSelfEvents or something like that, that sounds good to me too. When we make such a change, perhaps having an alias "ignoreSelf" for backward compatibility for a while would be a good way to go.